### DISCUSSION TITLE
Feature Request: Button to open DAX Studio using the current connection string from Tabular Editor

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1025] [Ideas]
**Is your feature request related to a problem? Please describe.**
I am always using Tabular Editor along with DAX Studio to edit and query models in the service. Currently this involves opening DS separately and connecting to the workspace, then finding and opening the same dataset.

**Describe the solution you'd like**
It would be nice if TE had a button that would open DAX Studio the current dataset, passing the workspace/dataset name to it.

[I have opened a feature request on the DAX Studio forum to open Tabular Editor as well](https://github.com/DaxStudio/DaxStudio/issues/936).

**Comment by otykier** [comment]
We are working on providing a better API for C# scripts to access documents (such as DAX queries) that are open in Tabular Editor. This way, it should be straightforward to create a macro that lets you open DAX Studio (or any other tool), pre-connected to the current model. Will let you know once this is possible.
